perfkitbenchmarker/publisher.py

```diff
@@ -185,6 +185,9 @@ def AddMetadata(self, metadata, benchmark_spec):
           for key, value in data_disk.metadata.iteritems():
             metadata[name_prefix + 'data_disk_0_' + key] = value
```

Comment:
Not related to this CR but the "key" in here could be an integer and so this should really be done with in the %s formatting way otherwise an error will be thrown about adding a string and an integer.